static/blog.html

```diff
@@ -0,0 +1,45 @@
+<html>
+<html>
+<head>
+</head>
+
+
+<body>
+
+<header>
+
+  <h1> Wang </h1>
+
+
+<nav>
+
+  <a href="/home"> Home </a>
+  <a href="/about"> About </a>
+  <a href="/projects"> Projects </a>
+  <a href="/blog"> Travel Journal </a>
+
+</nav>
+</header>
+
+<section class="travel_journal">
+  <br>
+  <h2> My Travel List </h2>
+
+  <!-- <article class="amsterdam"> -->
+    <h3> the Netherlands </h3>
+  <img src="./images/fifth.JPG">
```

Comment:
Use the `alt` attribute for images; it provides folks using screen readers and similar devices a description of the photo. Something like `<img src="./images/fifth.jpg" alt="something that describes the photo">`